src/app/game/Ludo/page.tsx

```
@@ -0,0 +1,322 @@
+          </div>
+        </div>
+
+        <div className={styles.rightPanel}>
+          {renderHomeArea('green')}
+          {renderHomeArea('yellow')}
+        </div>
+      </div>
+
+      <div className={styles.gameControls}>
+        <button className={styles.resetButton} onClick={resetGame}>
+          New Game
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)